esp/src/eclwatch/DynamicESDLMethodWidget.js

```diff
@@ -167,20 +167,12 @@ define([
                             __hpcc_parentName: null,
                             __hpcc_id: row.Name
                         });
-
-                        if (row.Attributes) {
-                            newRows.push({
-                                __hpcc_parentName: row.Name,
-                                __hpcc_id: row.Name + idx,
-                                Value: row.XML
-                            });
-                        } else {
-                            newRows.push({
-                                __hpcc_parentName: row.Name,
-                                __hpcc_id: row.Name + idx,
-                                Value: "<Method name=\"" + row.Name + "\"/>"
-                            });
-                        }
+                        
+                        newRows.push({
+                            __hpcc_parentName: row.Name,
+                            __hpcc_id: row.Name + idx,
+                            Value: row.XML
+                        });
```

Comment:
@miguelvazq I think you mentioned in an offline conversation that you're always getting back a xml entry, but I'm not sure that we can rely on that, it might be worthwhile to keep the "else" and insert the generated 
"<Method name="xyz"/>" portion